.github/workflows/ci-workflow.yml

```diff
@@ -36,6 +36,7 @@ env:
   MAVEN_PROFILES: >-
     -P contrib-check
     -P include-grpc
+    -P include-ranger
```

Comment:
Enabling the `include-ranger` profile in the build workflow should be unnecessary, since the profile activation occurs based on the detected architecture. Can this line be removed?